sdk/mediaservices/Azure.Media.Analytics.Edge/src/Customization/MediaGraphInstanceSetRequest.cs

```diff
@@ -5,20 +5,18 @@ namespace Azure.Media.Analytics.Edge.Models
 {
     public partial class MediaGraphInstanceSetRequest
     {
-        internal MediaGraphInstanceSetRequestBody InstanceBody;
+        private MediaGraphInstanceSetRequestBody _instanceBody;
```

Comment:
Do you need this field? Can it be a local?

Reply:
Fixed